src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -347,6 +347,14 @@ class AddressLookUpController extends AbstractController {
 
     response.redirect(pageRouting.nextUrl);
   }
+  private isPrincipalPlaceOfBusinessPage(pageType: AddressLookUpPageType): boolean {
```

Comment:
looks good, just add a blank line before the method between line 349 and 350. A line of whitespace before and after a method helps improve the readability.